jwks/jwks.go

```diff
@@ -13,13 +13,19 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/ONSdigital/dp-identity-api/models"
+	"github.com/ONSdigital/dp-identity-api/v2/models"
 	dphttp "github.com/ONSdigital/dp-net/v2/http"
 )
 
+// JWKSInt defines methods for managing JSON Web Key Sets (JWKS).
+// It provides domain-specific methods to retrieve and format JWKS data.
+//
 //go:generate moq -out mock/jwks.go -pkg mock . JWKSInt
```

Comment:
The godoc should be immediately before the definition. The `go:generate` should be separate so put it first.
```suggestion
//go:generate moq -out mock/jwks.go -pkg mock . JWKSInt

// JWKSInt defines methods for managing JSON Web Key Sets (JWKS).
// It provides domain-specific methods to retrieve and format JWKS data.
```